archinstall/lib/mirrors.py

```diff
@@ -5,7 +5,7 @@
 from .output import log
 
 
-def filter_mirrors_by_region(regions, destination='/etc/pacman.d/mirrorlist', tmp_dir='/root', *args, **kwargs):
+def filter_mirrors_by_region(regions, destination='/etc/pacman.d/mirrorlist', *args, **kwargs):
 	"""
```

Comment:
Nicely done, this old legacy code really needed to change. One question tho, is the tmp_dir really used or can it be dropped? :)